src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,65 @@
 package core.basesyntax;
 
-public class Car {
+/**
+ * Дано класс Car и класс Engine. Определите метод клон так чтобы клонирование было глубоким.
+ * Добавьте минимум 5 полей в каждый класс (Car and Engine).
+ * Не забудьте использовать двигатель в автомобиле.
+ */
+public class Car implements Cloneable {
+    private String nameCar;
+    private String modelCar;
+    private int maxSpedCar;
+    private int ageCar;
+    private int weightCar;
+    private Engine engine;
 
+    public Car(String nameCar, String modelCar, int maxSpedCar, int ageCar,
+               int weightCar, Engine engine) {
+        this.nameCar = nameCar;
+        this.modelCar = modelCar;
+        this.maxSpedCar = maxSpedCar;
+        this.ageCar = ageCar;
+        this.weightCar = weightCar;
+        this.engine = engine;
+    }
+
+    public Car clone() {
```

Comment:
we should override this method, not create new one

Reply:
done